### PR TITLE
Completed prep-hmtl-skeleton exercise.

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
prep-hmtl-skeleton - Jesus Alejandro Araujo

HTML_Skeleton

### CODE REVIEW

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What tags are necessary for a complete HTML Skeleton?
```
Comment (BrettAlbright) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:7]: make sure to answer all of these quiz questions

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Code looks solid Jesus, just make sure to answer the quiz questions

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md (hunk 1 above)
Comment (j-alejandro-araujo) [reply prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:7]: Quiz questions added, thank you! 

Asnwers_added

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**